### PR TITLE
completed ticket 7

### PR DESCRIPTION
**Pull request by DrawnInward**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -153,4 +153,51 @@ describe("GET /api/reviews", () => {
...
+  });
+  test("will give 400 when given an invalid ID", () => {
+    return request(app)
+      .post("/api/reviews/SELECT * FROM cards/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("bad request!");
+      });
+  });
+});
```
Comment (cjplatten) [inline __tests__/app.test.js:203]: I'd like to see few few more tests here! Think about what if there are missing required fields, incorrect username, valid but non-existent ID?

**Review by cjplatten — COMMENTED** [review]
A few comments to address!

**Review by mezzDavies — CHANGES_REQUESTED** [review]
Sorry Jonathan but you'll have to remove the .only on line 217 and resubmit the PR. We need to know that all your tests are passing to complete a review.

**Review by HannahBurks — APPROVED** [review]
Cool - feel free to merge 👏🏼

**Pull request merged**